Update Table names, Update Resource card with new fields

kate-darby opened the pull request:
Replaces resource table with service table, 
adds service type, 
Updates Resource Card with new field names

HaSistrunk commented on models/resources.ts:
```diff
@@ -34,16 +34,66 @@ export type Resource = {
+  attributes?: string[]
+  locations?: string[]
+  url?: string
+  phoneNumbers?: string
+  email?: string
+  Notes?: string
+  organizations?: string[]
+  'x-QualityController'?: string[]
+  'x-Researchers'?: string[]
+  'x-Contact'?: string 
+  'x-address'?: any[]
+  'x-Resources Lists'?: string[]
+  'Combined Taxonomy Terms'?: any
+  needFocus?: string[]
+  neighborhoodNames?: string[]
+  groupName?: any[]
+  assured_date?: string
+  assured_email?: string
+  application_process?: string
+  taxonomy_terms?: string[]
+  last_modified: string
+  alternative_name?: string
+  minimum_age?: number
+  maximum_age?: number
+  interpretation_services?: string[]
+  fees_description?: string
+  accreditations?: string
+  programs?: string[]
+  schedules?: string[]
+  contacts?: string[]
+  eligibility_description?: string
+  alert?: string
+  license?: string
+  Created: string
+  service_locations?: string[]
+  funding?: string[]
+  cost_options?: string[]
+  required_document?: string[]
+
+} 
```
```suggestion
export type Service = {
  id: string
  name: string
  status: ResourceStatus
  communityFocus?: string[]
  description?: string
  languages?: string[]
  url?: string
  phoneNumbers?: string
  email?: string
  organizations?: string[]
  'x-Resources Lists'?: string[]
  `x-address`?: any[]
  needFocus?: string[]
  groupName?: any[]
  assured_date?: string
  last_modified: string
  Created: string
} 
```

HaSistrunk replied:
We don't use/need many of these fields.

HaSistrunk commented on database/resources/needs.ts:
```diff
@@ -1,6 +1,9 @@
-export const selectAllNeeds = (filterFormula?: string): Promise<Need[]> => {
-  return ResourcesAirtableClient.selectAll<Need>('Ref - Need', filterFormula)
+export const selectAllNeeds = (filterFormula?: string): Promise<FieldSet[]> => {
+  // TODO: Figure out how to get all needFocus options or create a needs focus table
```
These value options are from the`taxonomy_terms` table, but only the ones with "MANYC Need" as the value in the "taxonomy" field. Is it possible to use that?

kate-darby commented:
> Ok! I got the Need Category filter working, but would appreciated a double check that I didn't do anything wonky in finding that solution.
> 
> I also updated the saved lists table to "x-Resources Lists", which it what it should be now (not "x-Resources to Add"). I also cleaned up a couple things and updated the RESOURCE_SEARCH_FIELDS names.

This looks great! I need to dig more into the querying, but I've been testing locally and everything looks good. I'm going to split out a second ticket for the work around adding and creating lists since this grew a little bigger than I would have liked and should have been split up.
